cram_pr2/cram_pr2_shopping_demo/CMakeLists.txt

```diff
@@ -0,0 +1,217 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(cram_pr2_shopping_demo)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
```

Comment:
Please remove these dependencies. CMakeLists is used for compiling C++ code and we have our ".asd" files for that. It is also used to generate executable files that you can run from terminal (not through emacs) but as you don't generate executables, you don't need this.

Reply:
It should just be 
find_package(catkin REQUIRED)

Reply:
Hmm, this is not going to work.
This is not even going to compile.
Please fix you CMakeLists.txt.
What I meant by my previous comment, is that you should not have
```
find_package(catkin REQUIRED COMPONENTS bla blablablabal)
```
but instead 
```
find_package(catkin REQUIRED)
```
You shouldn't have deleted catkin_package and all that stuff.